src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -360,49 +367,49 @@ Where `access_token` is the OAuth 2.0 access token of the user whose itinerary i
+|`Description`	|`string`	|The trip description. Maximum length: 512 characters.
+|`EndDateUtc`	|`dateTime`	|The end date of the trip, in UTC. Format: YYYY-MM-DDThh:mm:ss.
+|`IsPersonal`	|`boolean`	|Whether the trip is a Business or Leisure trip. Format: true/false.
+|`ProjectName`	|`string`	|The associated project name for the trip. Maximum length: 255 characters.
+|`StartDateUtc`	|`dateTime`	|The start date of the trip, in UTC. Format: YYYY-MM-DDThh:mm:ss.
+|`RuleViolations`	|`array`	|The list of rule violations associated with the itinerary. This parent element contains a `RuleViolation` child element for each associated rule violation.
+|`Status`	|`string`	|The status of the itinerary. One of the following: 0- Confirmed; 1- Ticketed by agent; 2- Canceled.
+|`Bookings`	|`array`	|A parent element that contains a `Booking` child element for each booking associated with this itinerary.
 
 ### Booking Element
 
 | Element Name | Data Type |Description
 | --------- | --------- | -------
-|Segments	|array	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, SAP Concur client,  third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs, clients, and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent elements.
-|Passengers	|array	|Contains a `Passenger` child element for each included passenger. For more information on the `Passengers` element, see [Create a New Trip](#postnt).
-|RecordLocator	|string	|The unique identifier for a booking. This is often six alphanumeric characters, but can have other formats depending on the booking source.
-|BookingSource	|string	|The name of the booking source for this booking. A booking source is a textual name the system uses to track where a booking took place.
-|DateModifiedUtc	|dateTime	|The date the booking was last modified, in UTC. Format: YYYY-MM-DDThh:mm:ss.
-|DateBookedLocal	|dateTime	|The date the booking was created, in the booking location's local time. Format: YYYY-MM-DDThh:mm:ss.
-|ItinSourceName	|string	|The itinerary source. Format: TravelSupplier.
-|PassengerCount	|integer	|The number of passengers included in the booking.
+|`Segments`	|`array`	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, SAP Concur client,  third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs, clients, and third-party developers**, the `Segments` element contains one or more `Air`, `Car`, `Hotel`, Di`ning, `Ride`, `Rail`, `Parking`, or `Travel` parent elements. **For TripLink suppliers**, the `Segments` element contains one or more `Air`, `Car`, `Hotel`, or `Ride` parent elements.
```

Comment:
Dining has a rogue backtick that is throwing everything off.

Reply:
Fixed